lib/wings/services/custom_queries/find_file_metadata.rb

```diff
@@ -33,7 +33,13 @@ def initialize(query_service:)
       #
       # @raise [Hyrax::ObjectNotFoundError]
       def find_file_metadata_by(id:, use_valkyrie: true)
-        find_file_metadata_by_alternate_identifier(alternate_identifier: id, use_valkyrie: use_valkyrie)
+        if id.to_s.starts_with?('fedora://')
+          find_file_metadata_by_alternate_identifier(alternate_identifier: id, use_valkyrie: use_valkyrie)
+        else
```

Comment:
this branch is for Valkyrie objects that need AF counterparts?

Reply:
that's right.